netbox/ipam/filtersets.py

```diff
@@ -1062,6 +1072,23 @@ def get_for_device(self, queryset, name, value):
     def get_for_virtualmachine(self, queryset, name, value):
         return queryset.get_for_virtualmachine(value)
 
+    def test_filter_interface_id(self, queryset, name, value):
```

Comment:
Is this function name right? `test_`?

Reply:
Yeah sorry, I neglected to commit my last set of changes before pushing.